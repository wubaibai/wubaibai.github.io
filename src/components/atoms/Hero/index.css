.hero {
	color: var(--lightBlue);
	font-family: var(--fontPrimary);
	font-size: 48px;
	font-weight: 500;
	line-height: 1.2;
	margin-top: 120px;
}

.job {
	display: flex;

	& > div:first-child {
		margin-right: 12px;
	}
}

@media (--mobile) {
	.hero {
		margin-top: 60px;
		font-size: 1.6rem;
	}
}
