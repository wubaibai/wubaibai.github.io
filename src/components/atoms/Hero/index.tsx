import React from 'react';

import style from './index.css';

const Hello: React.FC = () => (
	<div id="hero" className={style.hero}>
		<div id="hi" className={style.hi}>
			嗨！我是凱西，
		</div>
		<div className={style.position}>A Front-end Developer</div>
	</div>
);

export default Hello;
