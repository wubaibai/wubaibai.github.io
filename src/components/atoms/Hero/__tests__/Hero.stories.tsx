import React from 'react';
import { storiesOf } from '@storybook/react';

import Hero from 'components/atoms/Hero';

const stories = storiesOf('atoms/Hero', module);

stories.add('__interactive', () => <Hero />, { jest: 'Hero' });
