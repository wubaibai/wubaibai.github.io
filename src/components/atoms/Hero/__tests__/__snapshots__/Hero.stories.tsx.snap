// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`__interactive 1`] = `
<DocumentFragment>
  <div
    class="hero"
    id="hero"
  >
    <div
      class="hi"
      id="hi"
    >
      嗨！我是凱西，
    </div>
    <div>
      A Front-end Developer
    </div>
  </div>
</DocumentFragment>
`;
